plugin.php

```diff
@@ -1,190 +1,34 @@
-				<option style="padding-right:10px;" value="circles" <?php selected('circles', $instance['icon_set']); ?>><?php _e('Circles', 'social-profiles-widget'); ?></option>
-				<option style="padding-right:10px;" value="denim" <?php selected('denim', $instance['icon_set']); ?>><?php _e('Denim', 'social-profiles-widget'); ?></option>
-				<option style="padding-right:10px;" value="inside" <?php selected('inside', $instance['icon_set']); ?>><?php _e('Inside', 'social-profiles-widget'); ?></option>
-				<option style="padding-right:10px;" value="sketch" <?php selected('sketch', $instance['icon_set']); ?>><?php _e('Sketch', 'social-profiles-widget'); ?></option>
-			</select>
-		</p>
-
-		<p>
-			<label for="<?php echo $this->get_field_id('size'); ?>"><?php _e('Icon Size', 'social-profiles-widget'); ?>:</label>
-			<select id="<?php echo $this->get_field_id('size'); ?>" name="<?php echo $this->get_field_name('size'); ?>">
-				<option style="padding-right:10px;" value="24x24" <?php selected('24x24', $instance['size']); ?>><?php _e('Mini', 'social-profiles-widget'); ?> (24px)</option>
-				<option style="padding-right:10px;" value="32x32" <?php selected('32x32', $instance['size']); ?>><?php _e('Small', 'social-profiles-widget'); ?> (32px)</option>
-				<option style="padding-right:10px;" value="48x48" <?php selected('48x48', $instance['size']); ?>><?php _e('Large', 'social-profiles-widget'); ?> (48px)</option>
-			</select>
-		</p>
+ * NOTE: This plugin is released under the GPLv2 license. The images packaged with this plugin are the property
+ * of their respective owners, and do not, necessarily, inherit the GPLv2 license.
+ *
+ * @package social-profiles-widgets
+ */
 
-		<p><?php _e('Enter the URL(s) for your various social profiles below. If you leave a profile URL field blank, it will not be used.', 'social-profiles-widget'); ?></p>
+define( 'SOCIAL_PROFILES_WIDGET_DIR', plugin_dir_path( __FILE__ ) );
+define( 'SOCIAL_PROFILES_WIDGET_URL', plugins_url( '', __FILE__ ) );
 
-<?php
+if ( ! class_exists( 'Social_Profiles_Widget' ) ) {
+	require_once SOCIAL_PROFILES_WIDGET_DIR . '/includes/class-social-profiles-widget.php';
+}
 
-		foreach ( $this->spw_fields_array( $instance ) as $key => $data ) {
-			$url = isset( $instance[ $key ] ) ? $instance[ $key ] : '';
-			echo '<p class="wp-clearfix">';
-			printf( '<img style="float: left; margin-right: 3px;" src="%s" title="%s" />', $data['img_widget'], $data['img_title'] );
-			printf( '<label for="%s"> %s:</label><br />', esc_attr( $this->get_field_id($key) ), esc_attr( $data['title'] ) );
-			printf( '<input id="%s" name="%s" value="%s" style="%s" />', esc_attr( $this->get_field_id($key) ), esc_attr( $this->get_field_name($key) ), esc_url( $url ), 'width:65%;' );
-			echo '</p>' . "\n";
-		}
+add_action( 'widgets_init', 'social_profiles_widget_register' );
 
-	}
-}}
+/**
+ * Register widget.
+ */
+function social_profiles_widget_register() {
+		register_widget( 'Social_Profiles_Widget' );
```

Comment:
Looks like there's an extra tab line 33, plugin.php

Reply:
Should be resolved in the last commit @dreamwhisper :)